scripts/cli/cli_helper.py

```diff
@@ -1,24 +1,23 @@
 import argparse
 import json
-import os
 from datetime import datetime
 from os import environ
 from typing import List, NamedTuple, Optional
 
 from dateutil import parser, tz
-from linz_logger import get_log
 
-from scripts.files.files_helper import get_file_name_from_path
+from scripts.gdal.gdalinfo import GdalInfo, gdal_info, get_origin
+from scripts.tile.tile_index import TileIndexException, get_tile_name
 
 
 class TileFiles(NamedTuple):
     output: str
     input: List[str]
 
 
-def format_source(source: List[str]) -> List[TileFiles]:
+def get_tile_files(source: List[str], scale: int = 0) -> List[TileFiles]:
```

Comment:
The input parameter `source` is confusing in this function as it could be `["[{'output': 'CE16_5000_1001', 'input': ['s3://bucket/SN9457_CE16_10k_0501.tif']}]"]` or `['SN9457_CE16_10k_0501.tif', 'SN9457_CE16_10k_0502.tif']`.
I'm wondering if we should prepare the data separately so this function have only one kind of input?

Reply:
Also the input source of type `["[{'output': 'CE16_5000_1001', 'input': ['s3://bucket/SN9457_CE16_10k_0501.tif']}]"]` should probably not exist anymore as we would use `--from-file` with Argo.